src/components/Card/readme.md

```diff
@@ -210,3 +210,74 @@ function PriceCard(props) {
 </div>
 
 ```
+##### Card with Progress circular chart
+
+```js
+import React from 'react';
+import { Card, HelpText, ProgressCircular } from 'react-rainbow-components';
+import styled from 'styled-components';
+
+const StyledCard = styled(Card)`
+    width: 257px;
+    height: 105px;
+    display: flex;
+    justify-content: space-between;
+`;
+
+const StyledProgressCircular = styled(ProgressCircular)`
+    width: 80px;
+    height: 80px;
+`;
+
+const Subtitle = styled.h2`
+    font-family: 'Lato Light';
+    font-size: 16px;
+    color: #788497;}
```

Comment:
Should use a theme color to change them dynamically.

Reply:
also, put the `}` in a new line